docs/cse/integrations/integrate-cse-with-taxii-feed.md

```diff
@@ -17,15 +19,15 @@ To integrate CSE with a TAXII feed, you configure the URL of the TAXII provider
 
 ## Leveraging indicators in rules
 
-The integration allows you to enrich incoming Records with threat intel information, and leverage that information in CSE Rules. How does that work? CSE compares incoming Records with information from the threat feed. When there is a “match”, for instance when an IP address in a Record matches an IP address that the feed says is malicious, CSE adds relevant information to that Record. Because the threat intel information is persisted within Records, you can reference it downstream in both rules and search. The built-in rules that come with CSE will also automatically create a Signal for any Record with a match from your threat feed.To leverage the information in a rule, you can extend your custom rule expression, or add a [Rule Turning Expression](../rules/rule-tuning-expressions.md) to a built-in rule. For a more detailed explanation of how to use threat intel information in rules, see [Threat Intel](../rules/about-cse-rules.md) in the *About CSE Rules* topic.
+The integration allows you to enrich incoming Records with threat intel information, and leverage that information in CSE Rules. How does that work? CSE compares incoming Records with information from the threat feed. When there is a “match”, for instance when an IP address in a Record matches an IP address that the feed says is malicious, CSE adds relevant information to that Record. Because the threat intel information is persisted within Records, you can reference it downstream in both rules and search. The built-in rules that come with CSE will also automatically create a Signal for any Record with a match from your threat feed. To leverage the information in a rule, you can extend your custom rule expression, or add a [Rule Turning Expression](../rules/rule-tuning-expressions.md) to a built-in rule. For a more detailed explanation of how to use threat intelligence information in rules, see [Threat Intelligence](../rules/about-cse-rules/#threat-intelligence) in the *About CSE Rules* topic.
```

Comment:
```suggestion
The integration allows you to enrich incoming Records with threat intel information, and leverage that information in CSE Rules. How does that work? CSE compares incoming Records with information from the threat feed. When there is a “match”, for instance when an IP address in a Record matches an IP address that the feed says is malicious, CSE adds relevant information to that Record. Because the threat intel information is persisted within Records, you can reference it downstream in both rules and search. The built-in rules that come with CSE will also automatically create a Signal for any Record with a match from your threat feed. To leverage the information in a rule, you can extend your custom rule expression, or add a [Rule Turning Expression](/docs/cse/rules/rule-tuning-expressions) to a built-in rule. For a more detailed explanation of how to use threat intelligence information in rules, see [Threat Intelligence](/docs/cse/rules/about-cse-rules/#threat-intelligence) in the *About CSE Rules* topic.
```